src/main/java/courseWork/courseWork2/controller/JavaQuestionController.java

```diff
@@ -15,28 +16,22 @@ public class JavaQuestionController {
 
     public final QuestionService questionService;
 
-    public JavaQuestionController(QuestionService questionService) {
+    public JavaQuestionController(@Qualifier("java") QuestionService questionService) {
```

Comment:
```suggestion
    public JavaQuestionController(@Qualifier("javaExam") QuestionService questionService) {
```
название бина неверное